apps/pigment-css-next-app/next.config.js

```diff
@@ -96,14 +96,40 @@ theme.getColorSchemeSelector = (colorScheme) => {
   return `@media (prefers-color-scheme: ${colorScheme})`;
 };
 
+function innerNoop() {
+  return null;
+}
+function outerNoop() {
+  return innerNoop;
+}
+
 /**
  * @type {PigmentOptions}
  */
 const pigmentOptions = {
   theme,
   transformLibraries: ['local-ui-lib', '@mui/material'],
   sourceMap: true,
-  displayName: true,
+  overrideContext: (context) => {
```

Comment:
Prevent an error `Cannot redefine property toString`

Reply:
I'm not very familiar with the Pigment setup, why is this required in detail?

Is `displayName` no longer required?

Reply:
I'm not sure why @brijeshb42 removes the `displayName`, but for `overrideContext` it's a workaround for an error related to MUI System to make the app runnable (I will open a separate issue to discuss the options).